assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -121,10 +121,10 @@ const baseActions = {
 		invariant( isValidAccountSelection( accountID ), 'A valid accountID is required to select.' );
 
 		const registry = yield Data.commonActions.getRegistry();
-		registry.dispatch( STORE_NAME ).setAccountID( accountID );
-		registry.dispatch( STORE_NAME ).setPropertyID( '' );
-		registry.dispatch( STORE_NAME ).setInternalWebPropertyID( '' );
 		registry.dispatch( STORE_NAME ).setProfileID( '' );
+		registry.dispatch( STORE_NAME ).setInternalWebPropertyID( '' );
+		registry.dispatch( STORE_NAME ).setPropertyID( '' );
+		registry.dispatch( STORE_NAME ).setAccountID( accountID );
```

Comment:
It might help to add a clarifying comment here on that this order matters and why. Alternatively, this might be even a bit more error-proof by using the `setSettings` action and passing an object (with `accountID`, `propertyID`, `internalWebPropertyID`) to set all three values at the same time.

Reply:
Yep, `setSettings` is better. I have updated it to use that action.